integrations/spark-extensions/build.gradle.kts

```diff
@@ -36,6 +36,7 @@ dependencies {
   compileOnly("org.apache.spark:spark-core_${sparkScala.scalaMajorVersion}") { forSpark(sparkScala.sparkVersion) }
   compileOnly("org.apache.spark:spark-hive_${sparkScala.scalaMajorVersion}") { forSpark(sparkScala.sparkVersion) }
   implementation(nessieClientForIceberg())
+  implementation(nessieProject("nessie-notice"))
```

Comment:
LICENSE and NOTICE are included in the jar under `/META-INF/resources/`... why not directly under `META-INF` (e.g. Guava has it there)?

Reply:
The "regular" place is `META-INF/LICENSE` and `META-INF/NOTICE` (at least it's what some maven plugins like shade plugin are looking for).

Reply:
> META-INF/LICENSE and META-INF/NOTICE

Pretty bad for uber-jars - everything overwrites everything

Reply:
true, but Nessie has a specific binary NOTICE for uber jars, right?

Reply:
That's not what LICENSE + NOTICE are for.

Reply:
The "binary uber/container/..." distributables get theirs in /META-INF/resources/LICENSE.txt (and NOTICE.txt)